examples/ddoc/main.rs

```diff
@@ -30,15 +30,18 @@ impl DocFileLoader for SourceFileFetcher {
   fn load_source_code(
     &self,
     specifier: &str,
-  ) -> Pin<Box<dyn Future<Output = Result<String, DocError>>>> {
+  ) -> LocalBoxFuture<Result<(Syntax, String), DocError>> {
```

Comment:
I'm curious if `LocalBoxFuture` is required here... In Deno most of the futures are not `Send` and we do not use this `LocalBoxFuture`. Doesn't `Pin<Box<...>>` work? You can use `.boxed_local()` on the future to turn make type checker happy.

Reply:
`LocalBoxFuture` is just a type alias for `Pin<Box<dyn Future>>` - this was to satisfy a nightly clippy lint about types being too long.

Reply:
Fair enough; I'm gonna land this PR and cut a release of `deno_doc` soon.